packages/mattermost-redux/src/actions/helpers.ts

```diff
@@ -106,7 +106,7 @@ export function bindClientFunc({
 
 // Debounce function based on underscores modified to use es6 and a cb
 
-export function debounce(func: (...args: any) => unknown, wait: number, immediate: boolean, cb: () => unknown) {
+export function debounce(func: (...args: any) => unknown, wait: number, immediate?: boolean, cb?: () => unknown) {
```

Comment:
Do i have to change this in mattermost-redux

Reply:
Nope, this is the new home of `mattermost-redux` so that's perfect.